algo/algo_test.go

```diff
@@ -0,0 +1,29 @@
+package algo
```

Comment:
@codelingobot capture Files need to include license

Unless another license is specified, each file needs to include the apache license.